install/systemd.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: docs
+title: Run Airsonic with Systemd
+permalink: /docs/install/systemd/
+---
+##### Systemd
+
+Systemd is an init system for most linux systems. It allows one to run
+airsonic on boot.
+
+1. Setup dedicated airsonic user
+
+```
+useradd airsonic
+```
+
+2. Setup Airsonic data dir
+
+```
+mkdir /var/airsonic
+chown airsonic /var/airsonic
+```
+
+3. Download war
+
+```
+wget https://github.com/airsonic/airsonic/releases/download/v10.0.0/airsonic-jdbc-extra.war --output-document=/var/airsonic/airsonic.war
+```
+
+4. Setup systemd service
+
+```
+wget https://raw.githubusercontent.com/airsonic/airsonic/master/contrib/airsonic.service -O /etc/systemd/system/airsonic.service
```

Comment:
For people that want to copy paste, they can go to that address and copy from there. I did it this way to prevent clogging up the doc with an inlined file and because most people will want the easy way.